dev/build_release.py

```diff
@@ -529,6 +511,14 @@ def init_argument_parser(cls, parser):
         '--bintray_repo', default='',
         help='Publish to this bintray repo.\n'
              'This requires BINTRAY_USER and BINTRAY_KEY are set.')
+      parser.add_argument(
```

Comment:
jar repo shouldnt have a default.
Maybe this should be a property?

Is this jar repo write only or is it also read?
When other people run this script, they shouldnt publish to the official repo, but should be able to read form it rather than building everything themselves. But if they do build things, they should read those from their repo. So this should be for writing, and there should be a path set somewhere that this has precedence of the existing external path for jar files in this repo but not elsewhere.